src/game-loop/src/main-dude/states/MainDudeRunningLookingUpState.cpp

```diff
@@ -0,0 +1,117 @@
+#include "main-dude/states/MainDudeRunningLookingUpState.hpp"
+#include "main-dude/MainDude.hpp"
+#include "Input.hpp"
+
+#include "logger/log.h"
```

Comment:
No.

Reply:
My bad, I initially left them out and only kept locally, but they slipped through my sleepy fingers.